docs/src/components/InteractivePlayground/useClampPlayground/example.tsx

```diff
@@ -0,0 +1,159 @@
+interface ClampPlaygroundOptions {
+  lowerBound: number;
+  upperBound: number;
+  lowerSpringToValue: number;
+  upperSpringToValue: number;
+}
+
+interface Props {
+  options: ClampPlaygroundOptions;
+}
+export default function useClampPlayground({ options }): Props {
+  const toggle = useSharedValue(false);
+  const width = useSharedValue(100);
+
+  const config = {
+    duration: 25000,
+    dampingRatio: 0.075,
+  };
+
+  const clampedStyle = useAnimatedStyle(() => {
+    return {
+      width: withClamp(
+        { min: options.lowerBound, max: options.upperBound },
+        withSpring(width.value, config)
+      ),
+    };
+  });
+  const defaultStyle = useAnimatedStyle(() => {
+    return {
+      width: withSpring(width.value, config),
+    };
+  });
+
+  function renderExample(testedStyle, description, showClampMarkers) {
+    return (
+      <View>
+        <Text style={styles.text}>{description}</Text>
+        <View
+          style={{
+            width: FRAME_WIDTH + 2 * BORDER_WIDTH,
+            borderWidth: BORDER_WIDTH,
+            borderColor: VIOLET,
+          }}>
+          <View>
+            <View
+              style={[
+                styles.toValueMarker,
+                {
+                  width: options.lowerSpringToValue,
+                },
+              ]}
+            />
+            {showClampMarkers && (
+              <View
+                style={[
+                  styles.clampMarker,
+                  {
+                    width: options.lowerBound,
+                  },
+                ]}
+              />
+            )}
+          </View>
+          <Animated.View style={[styles.movingBox, testedStyle]} />
+          <View>
+            <View
+              style={[
+                styles.toValueMarker,
+                {
+                  marginTop: -CLAMP_MARKER_HEIGHT / 2,
+                  width: FRAME_WIDTH - options.upperSpringToValue,
+                  alignSelf: 'flex-end',
+                },
+              ]}
+            />
+            {showClampMarkers && (
+              <View
+                style={[
+                  styles.clampMarker,
+                  {
+                    marginTop: -100,
+                    width: FRAME_WIDTH - options.upperBound,
+                    alignSelf: 'flex-end',
+                  },
+                ]}
+              />
+            )}
+          </View>
+        </View>
+      </View>
+    );
+  }
+
+  return (
+    <View style={styles.container}>
+      {renderExample(clampedStyle, 'Clamped spring', true)}
+      {renderExample(defaultStyle, 'Default spring', false)}
+
+      <Button
+        title="toggle"
+        onPress={() => {
+          toggle.value = !toggle.value;
+          width.value = toggle.value
+            ? options.lowerSpringToValue
+            : options.upperSpringToValue;
+        }}
+      />
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    flexDirection: 'column',
```

Comment:
```suggestion
    flexDirection: 'column',
    alignItems: 'center',
    justifyContent: 'center',
```

lets center it a bit